src/st/e_stage_name.h

```diff
@@ -1,8 +1,8 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
 #include "stage.h"
 
-#ifndef CASTLE_FLAG_BANK
-#define CASTLE_FLAG_BANK 0
+#ifndef STAGE_FLAG
+#define STAGE_FLAG NULL_STAGE_FLAG
```

Comment:
Doesn't matter too much but maybe it makes more sense to make the default `OVL_EXPORT(STAGE_FLAG)` and override it to `NULL_STAGE_FLAG` for those specific overlays?

Reply:
It's tough to say without having the inverted castle headers figured out what the best option would be, the inverted castle doesn't have flags associated with going to an OVL for the first time like the normal castle.  I think for now leaving it as is will be the least effort option.

I have no real attachment either way, I can do whatever is needed.